timemachine/cpp/src/gpu_utils.cuh

```diff
@@ -83,3 +83,21 @@ double __device__ __forceinline__ rmul_rn(double a, double b) { return __dmul_rn
 float __device__ __forceinline__ radd_rn(float a, float b) { return __fadd_rn(a, b); }
 
 double __device__ __forceinline__ radd_rn(double a, double b) { return __dadd_rn(a, b); }
+
+// Faster ERFC approximation courtesy of Norbert Juffa. NVIDIA Corporation
```

Comment:
Is this derived from the `my_erfcf` definition in https://forums.developer.nvidia.com/t/optimized-version-of-single-precision-error-function-erff/40977/18 ? (or somewhere else where the applied optimizations might be documented?)

Reply:
Looks like there is some similarity. Don't have a good sense for the exact providence, but the papers cited in the link you posted might provide some insight. Will check them out.